frappe/utils/xlsxutils.py

```diff
@@ -61,13 +61,9 @@ def handle_html(data):
 	obj.ignore_links = True
 	obj.body_width = 0
 	value = obj.handle(h)
-	value = value.split('\n', 1)
-	value = value[0].split('# ',1)
-	if len(value) < 2:
-		return value[0]
-	else:
-		return value[1]
-
+	value = ", ".join(value.split('  \n'))
```

Comment:
Lets only add `value = ", ".join(value.split('  \n'))` after the `value = obj.handle(h)` to handle the br html tag.

Reply:
Meaning remove line 65? Had kept it as is because it already existed.